### PR TITLE
Add more information about overriding identity to documentation

### PR DESCRIPTION
**Pull request by PetrDlouhy**

### CODE REVIEW

**Comment by PetrDlouhy** [comment]
The docs expects, that #217 is merged.

**Review by bittner — CHANGES_REQUESTED** [review]
Looks good! – I hope you don't mind that I edit your changes so much.

File: docs/features.rst [hunk 1]
```diff
@@ -68,3 +68,23 @@ and is enabled by default.  To disable:
...
+
+or in the template::
+
+    {% with analytical_identity=request.user.uuid|default:None %}
+        {% analytical_head_top %}
+    {% endwith %}
+
+If you want to change the identity only for specific provider, use the
+`*_identity` context variable, where `*` is prefix specified on the provider
+page.
```
Comment (bittner) [inline docs/features.rst:90]:
```suggestion
If you want to change the identity only for specific provider use the
``*_identity`` context variable, where ``*``-prefix is the module name
of the specific provider.
```

**Comment by bittner** [comment]
I believe, we should also mention [context processors](https://docs.djangoproject.com/en/4.0/ref/templates/api/#writing-your-own-context-processors) as an option.

## Example

```python
# FILE: myproject/context_processors.py
from django.conf import settings

def get_identity(request):
    return {
        'analytical_identity': 'some-value-here',
    }
```

With your project's `settings.py` like this:

```python
# FILE: myproject/settings.py
TEMPLATES = [
    {
        'OPTIONS': {
            'context_processors': [
                'myproject.context_processors.get_identity',
            ],
        },
    },
]
```

That allows you as a developer to leave your view code untouched _and_ make sure that the variable is injected for _all_ templates.

**Comment by PetrDlouhy** [comment]
Hope all improvements are now in the PR.

File: docs/features.rst [hunk 2]
```diff
@@ -68,3 +68,54 @@ and is enabled by default.  To disable:
     Alternatively, add one of the variables to the context yourself
     when you render the template.
 
+
+
```
Comment (bittner) [inline docs/features.rst:72]:
```suggestion

```

Two empty lines should be enough.

File: docs/features.rst [hunk 3]
```diff
@@ -68,3 +68,54 @@ and is enabled by default.  To disable:
...
+or in the template:
+
+.. code-block:: django
+
+    {% with analytical_identity=request.user.uuid|default:None %}
+        {% analytical_head_top %}
+    {% endwith %}
+
+or in context proessor:
```
Comment (bittner) [inline docs/features.rst:93]:
```suggestion
or by implementing a context processor:
```

File: docs/features.rst [hunk 4]
```diff
@@ -68,3 +68,54 @@ and is enabled by default.  To disable:
...
+                ],
+            },
+        },
+    ]
+
+That allows you as a developer to leave your view code untouched and
+make sure that the variable is injected for all templates.
+
+If you want to change the identity only for specific provider use the
+``*_identity`` context variable, where ``*``-prefix is the module name
```
Comment (bittner) [inline docs/features.rst:120]:
```suggestion
``*_identity`` context variable, where the ``*`` prefix is the module name
```

**Review by bittner — CHANGES_REQUESTED** [review]
Great, thanks! Just another few corrections, then we're ready to merge.

**Review by bittner — APPROVED** [review]

**Comment by bittner** [comment]
Shoot! I should have squashed the commits... :worried:

**Comment by bittner** [comment]
Thanks for your contribution! :100:

**Pull request merged**